### ISSUE TITLE
AI Notebooks - US  launch (Beta)

### ISSUE THREAD

**Issue by mhrng** [ovh/public-cloud-roadmap#214]
## User story
As a customer from OVHcloud USA,
I would like to be able to launch managed Jupyter or VSCode notebooks with ease and CPU/GPU resources
so that
I will focus on my AI exploration jobs and not on the architecture/security layers.

## Acceptance criterias
The product should be 100% similar to OVHcloud.com, meaning at least:
- Ability to select JupyterLab or VSCode.
- Ability to select various frameworks such as Tensorflow, Pytorch, Huggingface, FastAI, ...
- Ability to add labels.
- Ability to link object storage data.
- Ability to authorize secured or public access (unsecure access).
- Quick link access to JupyterLab or VSCode.
- Observability tools (logs and metrics).
- CPU and GPU resources (model to be confirmed)
- Official support and SLA.

## Follow, vote and give your feedback
You can follow this task with the notification on the right tab.
Ask us anything here in the comments below, and vote with emojis for most requested items !
👍 to vote for this issue

## Discuss on Discord
Feel free to discuss with us on https://discord.com/invite/PwPqWUpN8G
